api/tests/tooling/airtable-builder/factory/build-thematic.js

```diff
@@ -2,6 +2,7 @@ module.exports = function buildThematic(
   {
     id,
     name,
+    nameEnUs,
```

Comment:
Je propose qu'on renomme nameEnUs en enUsName ce qui est plus correct en anglais. On pourrait également renommer name en frName également, ça aurait plus de sens. WDYT? 
```suggestion
    enUsname,
```

Reply:
Nous avons utiliser la même nomenclature que pour les autres champs. Il faudrait faire un travail de refacto global, peut être l'objet d'une PR à part entière.